package.json

```
@@ -23,10 +23,13 @@
   "scripts": {
     "prepare": "husky",
     "build": "npx hardhat compile",
+    "upgrade": "hardhat run ./scripts/upgrade.ts",
+    "upgrade-local-fork": "mkdir -p .openzeppelin/external-node && cp -r .openzeppelin/prod/. .openzeppelin/external-node && MANIFEST_DEFAULT_DIR=.openzeppelin/external-node npm run upgrade -- --network external-hardhat",
```

```suggestion
    "upgrade-local-fork": "MANIFEST_DEFAULT_DIR=.openzeppelin npm run upgrade -- --network external-hardhat",
```
with `.openzeppelin/unknown-134.json` in same original place

---

Edit: We can remove this task. See https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/51/files#r1907078813

---

Commit: 9fca4f0ff85589cc12b3a0c5ebd666eee539491d